pandas/core/generic.py

```diff
@@ -1936,15 +1936,22 @@ def keys(self):
         """
         return self._info_axis
 
-    def iteritems(self):
-        """
-        Iterate over (label, values) on info axis
+    def items(self):
+        """Iterate over (label, values) on info axis
 
-        This is index for Series, columns for DataFrame and so on.
+        This is index for Series and columns for DataFrame.
+
+        Returns
+        -------
+        Generator
         """
         for h in self._info_axis:
             yield h, self[h]
 
+    @Appender(items.__doc__)
+    def iteritems(self):
+        return self.items()
```

Comment:
Actually, I think both Series and DataFrame have their own implementation, so this one is nowhere used. So can rather be removed?

Reply:
I'll look into it,